src/platform/nrfconnect/CHIPPlatformConfig.h

```diff
@@ -69,3 +69,27 @@
 #ifndef CHIP_CONFIG_MAX_FABRICS
 #define CHIP_CONFIG_MAX_FABRICS 5
 #endif
+
+// Disable some of the too detailed log modules to save flash
+#define CHIP_CONFIG_LOG_MODULE_ExchangeManager_DETAIL 0
+#define CHIP_CONFIG_LOG_MODULE_ExchangeManager_PROGRESS 0
```

Comment:
Is this module used to print Matter messages with message counters etc? If so, I think these are one of the most important logs so I would leave them. Instead probably detail logs for DeviceLayer are not very useful.